R/thematic.R

```diff
@@ -155,6 +160,7 @@ thematic_theme <- function(bg = "auto", fg = "auto", accent = "auto",
     if (identical(accent, fmls$accent)) new$accent <- NULL
     if (identical(sequential, fmls$sequential)) new$sequential <- NULL
     if (identical(qualitative, fmls$qualitative)) new$qualitative <- NULL
+    if (identical(ggtheme, fmls$ggtheme)) new$ggtheme <- NULL
```

Comment:
Add a unit test to make sure `identical(formals(thematic_on), formals(thematic_theme))` (and ditto for `thematic_shiny()`, `thematic_rmd()`)